src/components/Track/Track.js

```diff
@@ -36,23 +36,23 @@ export class Track extends React.Component{
 After passing the track as an attirbute in TrackList.js I used the info
 here to render. Hoping each time it renders something different. */
             
-            <div className="Track">
-                <div className="Track-information">
+            <div className="track"> {/*a block level element*/}
+                <div className="track__information"> {/*an element level element */}
                     <h3>{trackInfo.name}</h3>
                     <p>{trackInfo.artist} | {trackInfo.album}</p>
                 </div>
                 {
                         isRemoval
                         
                             ? <a 
-                            className="Track-action" 
+                            className="track__action" 
                             onClick={this.removeTrack} 
                             /*href="www.#.com"*/>-</a>
 
-                            : <a className="Track-action"
+                            : <a className="track__action"
                                  onClick={this.addTrack} 
                                  /*href="www.#.com"*/>+</a>
-                    }
+                    } {/*an element level element */}

```

Comment:
something aint right in indentation here
I want to suggest to you to move and work with spaces and not tabs everywhere, and use 2 spaces. but this is alot to change now, so unless you dont mind. or next time. it is an opinionated thing, spaces vs tabs, keep it in mind. it might drive ppl crazy. I started using spaces since airbnb style guide choose them https://github.com/airbnb/javascript

Reply:
I fixed the indentations but did not converted tabs to double-spaces.. yet.
I'm not sure it's an effort well spent on this app, given that I would like to build at least another one from scratch where I will implement the knowlade you've passed on to me, Sensei 🍶  <-- Sake :)